en/appendices/3-1-migration-guide.rst

```diff
@@ -40,6 +40,24 @@ RoutesShell
 Controller
 ==========
 
+- The following Controller properties are now deprecated:
+
+  * layout
+  * view
+  * theme
+  * autoLayout
+  * viewPath
+  * layoutPath
+
+  Instead of setting these properties on your controllers, you should set them
+  on the view using methods with matching names::
+
+    // In a controller
+    $this->getView()->layout('advanced');
```

Comment:
don't you think it's worth showing an example of the old syntax, just to highlight the `"you should set them on the view using methods with matching names"`. I was not confused but I can already see people not getting the idea...

Appart from this, looks good